dapr-spring/pom.xml

```diff
@@ -0,0 +1,174 @@
+<project
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.dapr</groupId>
+    <artifactId>dapr-sdk-parent</artifactId>
+    <version>1.12.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>io.dapr.spring</groupId>
+  <artifactId>dapr-spring-parent</artifactId>
+  <packaging>pom</packaging>
+  <version>0.12.0-SNAPSHOT</version>
+  <name>dapr-spring-parent</name>
+  <description>SDK extension for Spring and Spring Boot</description>
+
+  <modules>
+    <module>dapr-spring-core</module>
+    <module>dapr-spring-data</module>
+    <module>dapr-spring-messaging</module>
+    <module>dapr-spring-boot-autoconfigure</module>
+    <module>dapr-spring-boot-starters/dapr-spring-boot-starter</module>
+    <module>dapr-spring-boot-testcontainers</module>
+  </modules>
+
+  <properties>
+    <springboot.version>3.2.6</springboot.version>
+    <testcontainers-dapr.version>0.12.0-SNAPSHOT</testcontainers-dapr.version>
+    <testcontainers-test.version>1.19.8</testcontainers-test.version>
+    <dapr.sdk.version>1.12.0-SNAPSHOT</dapr.sdk.version>
+    <dapr-workflows.sdk.version>0.12.0-SNAPSHOT</dapr-workflows.sdk.version>
```

Comment:
@artur-ciocanu I think in master now we have 
`<dapr.sdk.alpha.version>0.12.0-SNAPSHOT</dapr.sdk.alpha.version>` -> https://github.com/dapr/java-sdk/blob/master/pom.xml#L20C5-L20C69

Reply:
Fixed.